cmd/util/ledger/migrations/storage_reporter.go

```diff
@@ -1,193 +1,193 @@
 package migrations
 
-import (
-	"bufio"
-	"fmt"
-	"os"
-	"path"
-	"time"
-
-	"github.com/onflow/cadence/runtime/common"
-	"github.com/onflow/cadence/runtime/interpreter"
-	"github.com/rs/zerolog"
-
-	"github.com/onflow/flow-go/fvm/state"
-	"github.com/onflow/flow-go/ledger"
-	"github.com/onflow/flow-go/ledger/common/utils"
-	"github.com/onflow/flow-go/model/flow"
-)
-
-// iterates through registers keeping a map of register sizes
-// reports on storage metrics
-type StorageReporter struct {
-	Log       zerolog.Logger
-	OutputDir string
-}
```

Comment:
I'm not sure if this reporter works, anyway going to add proper reporters in a separate PR

Reply:
I think @janezpodhostnik maintained it well. Any reason for removing it if it still works?

Reply:
those changes are not merged, this code still not compile

Reply:
we might merge this back first https://github.com/onflow/flow-go/pull/1410/files

Reply:
Yes, this code does not compile in the current state.  https://github.com/onflow/flow-go/pull/1410 reenables them and adds some cleanup, but needs reviews.